src/test/ui/extern/extern-crate-multiple-missing.rs

```diff
@@ -0,0 +1,8 @@
+// If multiple `extern crate` resolutions fail each of them should produce an error
+extern crate bar; //~ ERROR can't find crate for `bar`
+extern crate foo; //~ ERROR can't find crate for `foo`
+
+fn main() {
+    foo::something();
+    bar::something();
```

Comment:
Could you add a comment telling that these lines using the unresolved crates above are not supposed to produce extra errors?